pkgs/development/tools/pnpm/fetch-deps/pnpm-config-hook.sh

```diff
@@ -44,11 +44,24 @@ pnpmConfigHook() {
 
     runHook prePnpmInstall
 
-    pnpm install \
+    if
+        ! pnpm install \
         --offline \
         --ignore-scripts \
         "${pnpmInstallFlags[@]}" \
         --frozen-lockfile
```

Comment:
nit: these should probably be indented

Reply:
I'm not sure whats the preferred indentation in bash inside a condition, I was also thinking of
```bash
    if
        ! pnpm install \
            --offline \
            --ignore-scripts \
            "${pnpmInstallFlags[@]}" \
            --frozen-lockfile
    then
```

or
```bash
    if pnpm install \
        --offline \
        --ignore-scripts \
        "${pnpmInstallFlags[@]}" \
        --frozen-lockfile; then
```

But one looks weirder than the other for me. Please comment the code in the specific style you'd like

Reply:
Hmm. I think the way it is now after your last push is fine.